spec/src/modules/autocomplete.js

```diff
@@ -77,7 +78,8 @@ describe('ConstructorIO - Autocomplete', () => {
         expect(res).to.have.property('sections').to.be.an('object');
         expect(res).to.have.property('result_id').to.be.an('string');
         expect(res.request).to.have.property(`ef-${Object.keys(testCells)[0]}`).to.equal(Object.values(testCells)[0]);
-        expect(requestedUrlParams).to.have.property(`ef-${Object.keys(testCells)[0]}`).to.equal(Object.values(testCells)[0]);
+        expect(requestedUrlParams).to.have.property(`ef-${Object.keys(testCells)[0]}`)
```

Comment:
Our ESLint was ignoring this although it's a long line. It started picking it up when I updated the eslintrc. Same with the other long lines in the PR